src/controller/CommissioningDelegate.h

```diff
@@ -581,6 +609,7 @@ class CommissioningParameters
         mAttestationNonce.ClearValue();
         mWiFiCreds.ClearValue();
         mCountryCode.ClearValue();
+        mTermsAndConditionsAcknowledgement.ClearValue();
```

Comment:
Why?  This is not an "ExternalBufferDependentValue", is it?

Reply:
You are correct. I removed the line. Now the mTermsAndConditionsAcknowledgement starts as empty Optional, and the `SetTermsAndConditionsAcknowledgement` can populate it, and re-populate it, but there is no way to return it back to Optional.